packages/frontend-shared/src/gql-components/topnav/TopNav.vue

```diff
@@ -22,7 +22,7 @@
     >
       <div class="font-medium">
         <ExternalLink
-          :href="`${releasesUrl}/tag/v${versions.latest.version}`"
+          :href="`https://on.cypress.io/changelog#${versions.latest.version.replaceAll('.', '-')}`"
```

Comment:
We could make a function in the script block to compose this string, taking `latest` or `current` as an argument, especially since we're doing a bit more work in there now.

Reply:
good point! should be updated now